python/tk_framework_desktopserver/shotgun/api_v2.py

```diff
@@ -70,6 +57,7 @@ def __init__(self, host, process_manager, wss_key):
         self._host = host
         self._engine = sgtk.platform.current_engine()
         self._wss_key = wss_key
+        self._logger = sgtk.platform.get_logger("api-v2")
```

Comment:
we should avoid creating and passing loggers around. Instead, the pattern should be that whenever we need a logger, we create one (usually via `__name__` and [get_logger](http://developer.shotgunsoftware.com/tk-core/platform.html#sgtk.platform.get_logger). Instead of holding on to this logger like this, could we just put a `logger = sgtk.platform.get_logger(__name__)` at the top of the file? At least that's a step closer to where we want to be logging wise. 

It looks like the `self.logger` property is called from within the class itself a lot (weird pattern! - this old code is a little odd!) - we could switch all that round to just call `logger` instead. Is the `logger` property used by external classes as well? We should refactor that or put on the list of things to refactor in that logging ticket.